Composer/packages/client/src/pages/botProject/AppIdAndPassword.tsx

```diff
@@ -107,7 +111,11 @@ export const AppIdAndPassword: React.FC<AppIdAndPasswordProps> = (props) => {
       <div css={title}>{formatMessage('Microsoft App ID')}</div>
       <div css={subtext}>
         {formatMessage.rich(
-          'A Microsoft App ID is required for your local Azure resources. If you’ve created an App ID already, you can add here. If not, your App ID and secret will be created when you provision resources for this bot. <a>Learn more.</a>',
+          'A Microsoft App ID is required for your local Azure resources.' +
```

Comment:
use backtick ``

Reply:
For line continuation or for string templating?  (formatMessage.rich can take a regular string).  I moved it to be a single line.